app/src/main/java/org/kiwix/kiwixmobile/views/PageBottomTabLayoutSnackBarBehavior.java

```diff
@@ -0,0 +1,60 @@
+package org.kiwix.kiwixmobile.views;
+
+import android.content.Context;
+import android.os.Build;
+import android.support.design.widget.CoordinatorLayout;
+import android.support.design.widget.CoordinatorLayout.Behavior;
+import android.support.design.widget.Snackbar;
+import android.support.v4.view.ViewCompat;
+import android.util.AttributeSet;
+import android.view.Gravity;
+import android.view.View;
+
+public class PageBottomTabLayoutSnackBarBehavior extends Behavior<View> {
+
+    public PageBottomTabLayoutSnackBarBehavior() {
```

Comment:
Please use 2 space indents. Otherwise this looks good.

Reply:
Fixed :)